src/symbolic.jl

```diff
@@ -540,30 +497,31 @@ function _subs(ex::Expr, d::Associative, funcs::Set{Symbol})
     changed = false
 
     for (i, arg) in enumerate(ex.args)
-        new_arg, arg_changed = _subs(arg, d, funcs)
+        new_arg, arg_changed = _subs(arg, d, funcs, shiftit)
         out_args[i] = new_arg
         changed = changed || arg_changed
     end
 
     out = Expr(ex.head)
     out.args = out_args
     out, changed
+
 end
 
 """
 ```julia
 subs(ex::Union{Expr,Symbol,Number}, from, to::Union{Symbol,Expr,Number}, funcs::Set{Symbol})
 ```
 
-Apply a substituion where all occurances of `from` in `ex` are replaced by `to`.
+Apply a substitution where all occurances of `from` in `ex` are replaced by `to`.
 
 Note that to replace something like `x(1)` `from` must be the canonical form
 for that expression: `(:x, 1)`
 """
 function subs(ex::Union{Expr,Symbol,Number}, from,
               to::Union{Symbol,Expr,Number},
               funcs::Set{Symbol})
-    _subs(ex, Dict(from=>to), funcs)[1]
+    _subs(ex, Dict(from=>to), funcs, shiftit)[1]
```

Comment:
Does `shiftit` exist in this scope?

Reply:
Again, _subs shouldn't do shifts any more.